### PR TITLE
[Page Header] New page header for notebooks and UI updates

### PR DESCRIPTION
**Pull request by TackAdam**
### Description
1. New page header for notebooks and UI updates
2. Updates some of the cypress testing (Following PR to finish)
3. Bugfix for applications(one line change to handle mode not selected on entry)
4. Disable Query assist flaky test
5. Add back gettingstartedID to plugin.tsx

New Navigation before:
Screenshot 2024-08-27 at 10 50 19 AM
Screenshot 2024-08-27 at 10 50 30 AM

New navigation after:
Screenshot 2024-08-27 at 10 17 27 AM
Screenshot 2024-08-27 at 10 07 25 AM
Screenshot 2024-08-27 at 10 07 41 AM
Screenshot 2024-08-27 at 10 10 22 AM
Screenshot 2024-08-27 at 10 10 37 AM

Old Navigation before:
Screenshot 2024-08-27 at 10 00 29 AM
Screenshot 2024-08-27 at 10 00 44 AM
After changes: 
Screenshot 2024-08-27 at 10 18 02 AM
Screenshot 2024-08-27 at 9 58 45 AM
Screenshot 2024-08-27 at 9 59 02 AM
Screenshot 2024-08-27 at 9 59 18 AM
Screenshot 2024-08-27 at 3 32 41 PM

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by TackAdam** [comment]
Screenshot 2024-08-27 at 3 34 20 PM

Icon size adjusted to match reporting.

File: .cypress/integration/notebooks_test/notebooks.spec.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ describe('Testing paragraphs', () => {
     cy.get('a[data-test-subj="createNotebookPrimaryBtn"]').click();
     cy.get('input[data-test-subj="custom-input-modal-input"]').focus().type(TEST_NOTEBOOK);
     cy.get('button[data-test-subj="custom-input-modal-confirm-button"]').click();
-    cy.get('div[data-test-subj="notebookTitle"]').contains(TEST_NOTEBOOK).should('exist');
+    cy.get('h3.euiTitle').contains(TEST_NOTEBOOK).should('exist');
```
Comment (sumukhswamy) [inline .cypress/integration/notebooks_test/notebooks.spec.js:138]: any other placeholder to test this similar to a data-test-subj? might fail if the ui changes

File: public/components/trace_analytics/components/dashboard/mode_picker.tsx [hunk 2]
```diff
@@ -13,14 +13,14 @@ const labels = new Map([
...
+  modes: Array<{
     id: string;
     title: string;
-  }[];
+  }>;
   selectedMode: TraceAnalyticsMode;
   setMode: (mode: TraceAnalyticsMode) => void;
 }) {
-  const { modes, selectedMode, setMode } = props;
+  const { modes = [], selectedMode, setMode } = props;
```
Comment (sumukhswamy) [inline public/components/trace_analytics/components/dashboard/mode_picker.tsx:23]: any reason modes is empty here? or is it a linting issue?

Comment (TackAdam) [reply public/components/trace_analytics/components/dashboard/mode_picker.tsx:23]: Mode is not selected when entering from applications, needs to handle the empty state properly for it to render.

File: .cypress/integration/notebooks_test/notebooks.spec.js (hunk 1 above)
Comment (TackAdam) [reply .cypress/integration/notebooks_test/notebooks.spec.js:138]: With moving the tittle from the div cypress was unable to find the data-test-subj attached to it. Will dig in deeper for a better solution when re-writing the test in follow-up PR.

**Comment by TackAdam** [comment]
Fixed eui call out for upgrade to span full page 
Screenshot 2024-08-28 at 11 58 09 AM

**Review by sumukhswamy — APPROVED** [review]

**Review by ps48 — APPROVED** [review]

File: public/components/notebooks/components/notebook.tsx [hunk 3]
```diff
@@ -50,17 +52,13 @@ import {
...
+import { HeaderControlledComponentsWrapper } from '../../../../public/plugin_headerControl';
+import { coreRefs } from '../../../framework/core_refs';
 
-const pageStyles: CSS.Properties = {
-  float: 'left',
-  width: '100%',
-  maxWidth: '1500px',
+const newNavigation = coreRefs.chrome?.navGroup.getNavGroupEnabled();
+
+const panelStyles: CSS.Properties = {
```
Comment (ps48) [inline public/components/notebooks/components/notebook.tsx:60]: Not a blocker: not a fan of custom css when we could have an EuiSpacer

File: public/components/notebooks/components/paragraph_components/para_input.tsx [hunk 4]
```diff
@@ -133,40 +133,45 @@ export const ParaInput = (props: {
...
+                    setSelectableError(false);
+                    setIsModalOpen(true);
+                  }}
+                >
+                  Browse
+                </EuiSmallButton>
+              </EuiFlexItem>
+            </EuiFlexGroup>
           </EuiFlexItem>
+          <EuiFlexItem grow={true} />
```
Comment (ps48) [inline public/components/notebooks/components/paragraph_components/para_input.tsx:170]: Not sure why is this needed?

Comment (TackAdam) [reply public/components/notebooks/components/paragraph_components/para_input.tsx:170]: With: 
Screenshot 2024-08-28 at 2 20 05 PM
Without: 
Screenshot 2024-08-28 at 2 21 41 PM
Certain page sizes caused it to collapse to the left and not maintain the look of the rest of the page without the flex item.
